resources/nheko.appdata.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2018 Vitaly Zaitsev <vitaly@easycoding.org> -->
+<component type="desktop">
+  <id>nheko.desktop</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPLv3+ and Public Domain and MIT and Boost and Apache License and CC-BY-4.0</project_license>
+  <name>nheko</name>
+  <summary>Desktop client for the Matrix protocol</summary>
+  <description>
+    <p>
+      The motivation behind the project is to provide a native desktop app
+      for Matrix that feels more like a mainstream chat app (Riot,
+      Telegram etc) and less like an IRC client.
```

Comment:
Feels/Sounds like matrix/nheko is a IRC protocol which is isn't. Maybe leave the IRC part out as neither nheko nor Matrix directly has something to do with IRC (and yes it has a bridge and yes both are chat programs)

Reply:
I got this information from Readme.md.

Reply:
Hm ok :) Still leaving it here maybe for mujx to read it (didn't compare to the nheko README.)

Reply:
I agree with @MTRNord. Could you use the description from https://github.com/mujx/nheko/blob/master/.ci/linux/create-packages.sh#L55

Reply:
@mujx Description need to be longer than one row.

Reply:
Then I guess you can stop at `chat app`.

Reply:
@mujx Fixed.